src/mono-runtimes/mono-runtimes.csproj

```diff
@@ -17,7 +17,8 @@
   <PropertyGroup>
     <BuildDependsOn>
       ResolveReferences;
-      _BuildUnlessCached
+      _BuildUnlessCached;
+      _Build;
```

Comment:
I don't think that's necessary: `_BuildUnlessCached` will call `ForceBuild` - unless cached - which will call `_Build`.